### ISSUE TITLE
Missing status information: EPUB Accessibility 1.0 and EPUB Accessibility Techniques 1.0

### ISSUE THREAD

**Issue by murata2makoto** [w3c/epub-specs#944]
EPUB Accessibility 1.0 and EPUB Accessibility Techniques 1.0 lack status information.

I don't think that we should revise published recommendations without any notice.  I vaguely remember that some W3C recommendations state that some minor changes were made and the URLs did not change.

**Comment by mattgarrish** [comment]
It looks like we missed a change in the template. I added the status sections during 3.1, but we probably were using the old format in a google doc while writing these and translated them to docbook verbatim to what was there.

Closing this issue, as status sections are auto-generated in w3c so the next update will have one.
